lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,69 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+                if ($('.accordion').length) {
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                navigation.checkForCurrent();
+                navigation.listenForClick();
+            },
+
+            checkForCurrent: function() {
+                var active = $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') ? $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') : 0,
+                    activeSection = $('.accordion .button-chapter:eq(' + active + ')');
+
+                navigation.closeAccordions();
+                navigation.openAccordionSection(activeSection);
+            },
+
+            listenForClick: function() {
+                $('.accordion').on('click', '.button-chapter', function(event) {
+                    navigation.closeAccordions(event.currentTarget);
+                    navigation.openAccordionSection(event.currentTarget);
+                });
+            },
+
+            closeAccordions: function(button) {
+                var menu = $(button).next('.chapter-content-container').find('.chapter-menu');
+                console.log(button, menu);
+
+                $('.accordion .button-chapter').each(function(event) {
+                    var el = $(this);
```

Comment:
For clarity can you please pass the element as the second parameter in the each function and use that instead of 'this'.